### PR TITLE
Feat/current user widget

### PR DESCRIPTION
**Pull request by svevang**
Sets up the rails asset pipeline and includes the id user widget + some basic styles.

### CODE REVIEW

File: config/initializers/assets.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.precompile << %w(prx_auth-rails_manifest.js)
```
Comment (cavis) [inline config/initializers/assets.rb:1]: What happens on rails apps without the asset pipeline?  (CMS / Feeder come to mind).

File: app/assets/images/prx_auth-rails/user.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (cavis) [inline app/assets/images/prx_auth-rails/user.svg:1]: I guess someday (for the "global nav" project), we'd want to move all assets/styles into ID itself.  But it is not this day!

**Review by cavis — COMMENTED** [review]
Just a question about the asset pipeline.  Otherwise looks good!

File: config/initializers/assets.rb (hunk 1 above)
Comment (svevang) [reply config/initializers/assets.rb:1]: Builds and boots fine.  I see that assets are disabled globally in e.g. cms, so this directive is a noop.

**Review by cavis — APPROVED** [review]
👍 it is!

**Pull request merged**